addons/hr_holidays/models/mail_message_subtype.py

```diff
@@ -28,7 +28,7 @@ def _update_department_subtype(self):
                 department_subtype = self.create({
                     'name': subtype.name,
                     'res_model': 'hr.department',
-                    'default': subtype.default or False,
```

Comment:
Why this change ?

Reply:
Because Geoffrey don't want to have this type subscribed by defaut :)